keyboard/README.md

```diff
@@ -22,13 +22,15 @@ fun hideKeyboard(builder: KeyboardManager.Builder) {
 
 The `KeyboardManager.Builder` and `KeyboardHostingView` are provided on the platform.
 
-On Android the builder uses a `LifecycleManagerObserver` (see Architecture), where the keyboardHostingView is any resource Id for a `View` attached to the `Activity` bound to the manager.
+On Android the builder uses a `LifecycleManagerObserver` (see Architecture) unless it is a member of a `ViewModel`, where the keyboardHostingView is any resource Id for a `View` attached to the `Activity` bound to the manager.
+
+You can use the `AppCompatActivity.hudBuilder` convenience method to get a builder that is valid during the lifespan of the Activity it belongs to.
```

Comment:
Small typo, you probably meant `AppCompatActivity.keyboardManagerBuilder()` here instead

Reply:
dec4f00